intl/msg_hash_us.h

```diff
@@ -8243,6 +8463,10 @@ MSG_HASH(
    MENU_ENUM_SUBLABEL_VIDEO_SHADER_PRESET,
    "Load a shader preset. The shader pipeline will be automatically set-up."
    )
+MSG_HASH(
+   MENU_ENUM_LABEL_HELP_VIDEO_SHADER_PRESET,
+   "Load Shader Preset. Load a shader preset directly. The shader menu is updated accordingly.\nIf the preset uses scaling methods which are not simple, (i.e. source scaling, same scaling factor for X/Y), the scaling factor displayed in the menu might not be correct."
```

Comment:
Possible bad inheritance here: that first line seems to have no reason of being there.

Reply:
Noted, same comment as previous.

Reply:
Surrounding a pair of parantheses with commas? Should be fixed.
Also, are "source scaling" and "same scaling factor" simple or not? They do sound simple to me, but it looks like they are meant to be examples for "not simple" methods? Example is unclear.